conf/openmetadata.yaml

```diff
@@ -138,6 +138,18 @@ logging:
       timeZone: UTC
       maxFileSize: 50MB
 
+
+applications:
+  appsPrivateConfiguration:
+    - name: SlackApplication
+      preview: ${SLACKAPPLICATION_PREVIEW:-false}
+      parameters:
+        clientId: ${SLACK_APP_CLIENT_ID:-"7199163258083.7289624672561"}
```

Comment:
let's remove these and put into slackApp.tml , without the secrets

Reply:
also can you confirm if the secret is needed, or this can be done on runtime on initiating the flow

Reply:
Checking.

Reply:
ClientSecret is required as they are used to identify and authenticate the application when interacting with Slack's API.